tests/lib/tools/vmcli/setup.cfg

```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = vmcli
+
+[options]
+# TODO bump?
+python_requires = >= 3.8
+packages =
+    vmcli
+
+install_requires =
+    telnetlib3
```

Comment:
this has python packaging only so that we'll get auto install of telnetlib3 when installing the tool with pip